src/components/DataSession.vue

```diff
@@ -26,6 +27,39 @@ function addOperation(operationDefinition) {
     .catch(error => console.error('Error:', error));
 }
 
+const getImages = async () => {
+  try {
+    const response = await fetch ('http://127.0.0.1:8000/api/datasessions/', {
```

Comment:
I wonder if we can pass the data session ID down as a prop to this component so that we can make the API query specifically for the selected datasession?

Reply:
Or I guess somehow keep track of the selected data session in the store so we know which API resource we're looking for